config/_default/languages.en.toml

```diff
@@ -1,7 +1,7 @@
 languageCode = "en"
 languageName = "English"
 weight = 1
-title = "Developer Portal"
+title = "Espressif Developer Portal"
```

Comment:
@pedrominatel Suggest adding company name to website title.

Reply:
Agree.